### PR TITLE
Buffs mech disabler

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

Mech disabler now is works like a shotgun, shooting 5 weak disabler beams(15 damage, the normal disabler beam has 30). The energy drain is increased to 100 from 30. Has a cooldown of 1.5 seconds.

The exact numbers(weapon scatter, damage, amount of projectiles) can be discussed.

## Why It's Good For The Game

When you are trying to nonlethally deal with someone, a normal security officer with their roundstart available equipment is more effective then an expensive lategame mech, due to that fact that it is really slow, cant use stunbattons and even if you manage somehow to stamcrit a person with a mech disabler you still need some time to get out in order to handcuff them. So mech disablers are useless, especially compared with other mech weapons, like rocket launchers and ballistic guns. This PR makes the mech disabler not trash, so roboticists would have a wider choice of useful weapons when making a mech.

## Changelog

:cl:
balance: Mech disabler now works as a shotgun, shooting 5 weak disabler beams at one time.
balance: Mech disabler now uses 100 energy instead of 30, and it's attack cooldown increased to 1.5 seconds instead of 0.8
/:cl:

### CODE REVIEW

**Comment by CRITAWAKETS** [comment]
honestly i'd argue this makes it weaker since if you're using a disabler it's likely your enemy's on the run

a BIG disabler beam would probably be better

**Comment by SuperSlayer0** [comment]
> honestly i'd argue this makes it weaker since if you're using a disabler it's likely your enemy's on the run
> 
> a BIG disabler beam would probably be better

This covers more area so you are more likely to hit your target rather then if you were shooting single beams

**Comment by DaydreamIQ** [comment]
Personally I think a machine gun disabler would be pretty fun too, but its a step in the right direction considering it was literally better to just use a clusterbang launcher to stamcrit someone

**Comment by SuperSlayer0** [comment]
> How tight is the spread? Can we see?

I will post an image/video when I will be able to

**Comment by SuperSlayer0** [comment]
Image of the mech disabler shot
image

File: code/modules/vehicles/mecha/equipment/weapons/weapons.dm [hunk 1]
```diff
@@ -91,12 +91,14 @@
 	harmful = TRUE
 
 /obj/item/mecha_parts/mecha_equipment/weapon/energy/disabler
-	equip_cooldown = 8
+	equip_cooldown = 1.5 SECONDS
 	name = "\improper CH-DS \"Peacemaker\" disabler"
 	desc = "A weapon for combat exosuits. Shoots basic disablers."
```
Comment (tralezab) [inline code/modules/vehicles/mecha/equipment/weapons/weapons.dm:96]: Amend this, it no longer shoots basic disablers

**Review by tralezab — CHANGES_REQUESTED** [review]
nitpick, but good change

File: code/modules/vehicles/mecha/equipment/weapons/weapons.dm (hunk 1 above)
Comment (SuperSlayer0) [reply code/modules/vehicles/mecha/equipment/weapons/weapons.dm]: Done

**Pull request merged**